src/Microsoft.ML.ImageAnalytics/ExtensionsCatalog.cs

```diff
@@ -129,7 +129,7 @@ public static ImageLoadingEstimator LoadImages(this TransformsCatalog catalog, s
         /// </remarks>
         /// <param name="catalog">The transform's catalog.</param>
         /// <param name="imageFolder">Folder where to look for images.</param>
-        /// <param name="type">Image type - VectorDataView type or ImageDataViewType. Defaults to ImageDataViewType if not specified or null.</param>
+        /// <param name="type">Image type flag - If true loads image as a VectorDataView type else loads image as ImageDataViewType. Defaults to ImageDataViewType if not specified or false.</param>
```

Comment:
We should come up with a better name than `type` for this parameter. `useImageType` or similar.

Reply:
useImageType sounds good. I will change that.